new/INS1013FAG.xml

```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS1013FAG" xml:lang="en" type="ins">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Fārṭā ʾAdiko Gabrǝʾel</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2024-04-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church" subtype="institution">
+                    <placeName xml:id="n1" xml:lang="gez">ፋርጣ፡ አዲኮ፡ ገብርኤል፡</placeName>
+                    <placeName corresp="#n1" xml:lang="en">Fārṭā ʾAdiko Gabrǝʾel</placeName>
+                    <placeName xml:id="n2" xml:lang="gez">ፋርጣ፡ ዓዲኮ፡ ገብርኤል፡</placeName>
+                    <placeName corresp="#n2" xml:lang="en">Fārṭā ʿĀdiko Gabrǝʾel</placeName>
+                    <country ref="LOC3010Ethiop">Ethiopia</country>
+                    <region notBefore="1991" type="RegionalState" ref="LOC1426Amhara">ʾAmharā</region>  
+                    <settlement ref="LOC3056Farta" type="warada">waradā Fārṭā</settlement>
```

Comment:
`+` qabale Adiqo

Reply:
I can create a place record for it. Is there anywhere an official record of the administrative subunits? Maybe creating records for all of them could be a Hiwi job (if a comprehensive list exists)?

Reply:
Yes, this might be useful, though some considerations should be made in advance, and some research as well. The administrative division is constantly changing. For more recent time it is getting complex.  The most stable seems to be the list of waradas which more or less coincides with many of the known historical regions, and remains stable more or less (even though their administrative affiliation may change); tabiyas - perhaps the same, but to find a full list of tabiyas from the recent time is difficult (or one has to search for very well).